src/Owin.Security.Providers.Dropbox/DropboxAuthenticationHandler.cs

```diff
@@ -65,13 +66,13 @@ public DropboxAuthenticationHandler(HttpClient httpClient, ILogger logger)
                 var redirectUri = requestPrefix + Request.PathBase + Options.CallbackPath;
 
                 // Build up the body for the token request
-                var body = new List<KeyValuePair<string, string>>
+                var body = new Dictionary<string, string>
                 {
-                    new KeyValuePair<string, string>("grant_type", "authorization_code"),
-                    new KeyValuePair<string, string>("code", code),
-                    new KeyValuePair<string, string>("redirect_uri", redirectUri),
-                    new KeyValuePair<string, string>("client_id", Options.AppKey),
-                    new KeyValuePair<string, string>("client_secret", Options.AppSecret)
+                    { "grant_type", "authorization_code" },
+                    { "code", code },
+                    {"redirect_uri", redirectUri },
+                    { "client_id", Options.AppKey },
```

Comment:
This introduces c#6 features in a project that is all c#5. I'm not against it, I just need a moment to think if its the right thing to do for us.

Reply:
yes this is fine. I have filed #217 as a follow up.

Reply:
That's fine. It was more done as a point of laziness and the way I read code. No functional change so feel free to ignore it.